src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,11 @@
+import styles from './Loading.module.scss';
+import logo from '../../assets/Temp_Roadwatch_logo.svg';
+
+export function Loading(): JSX.Element {
+  return (
+    <div className={styles['Loading__container']}>
+      <img src={logo} alt="Loading..." className={styles['Loading__logo']} />
```

Comment:
The `alt` text should be a bit more descriptive. Maybe you could say "Loading indicator" or "Loading indicator displaying the Roadwatch logo."

For more information on this please check out this article: https://wcag.com/blog/good-alt-text-bad-alt-text-making-your-content-perceivable/

Reply:
I ended up using "Loading indicator displaying the Roadwatch logo."